### PR TITLE
Add python script to generate the HelixPreCommands script that will set up the environment

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

File: scripts/benchmarks_ci.py [hunk 1]
```diff
@@ -250,8 +251,6 @@ def __main(args: list) -> int:
                 target_framework_moniker = micro_benchmarks.FrameworkAction.get_target_framework_moniker(framework)
                 commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli)
                 source_timestamp = dotnet.get_commit_date(framework, commit_sha, args.cli_repository)
-                os.environ['PERFLAB_HASH'] = commit_sha
-                os.environ['PERFLAB_BUILDTIMESTAMP'] = source_timestamp
```
Comment (billwert) [inline scripts/benchmarks_ci.py]: does it make sense to just roll back the entire change that brought this in at this point? (maybe it doesn't matter.)

File: scripts/ci_setup.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        default=[],
+        help='Configurations used in the build in key=value format'
+    )
+
+    return parser
+
+
+def __process_arguments(args: list):
+    parser = ArgumentParser(
+        description='Tool to run .NET micro benchmarks',
```
Comment (billwert) [inline scripts/ci_setup.py]: is this copied from somewhere? Doesn't seem like the right description.

File: scripts/dotnet.py [hunk 3]
```diff
@@ -303,23 +308,25 @@ def run(self,
...
     if not groups:
         raise ValueError("Unknown target framework: {}".format(framework))
 
     FrameworkVersion = namedtuple('FrameworkVersion', ['major', 'minor'])
     version = FrameworkVersion(int(groups.group(1)), int(groups.group(2)))
 
+    return version
+
+def get_sdk_path(version: str, dotnet_path: str = None) -> str:
+    """Gets the dotnet Host version from the `dotnet --info` command."""
```
Comment (billwert) [inline scripts/dotnet.py]: this seems like the wrong docstring?

File: scripts/ci_setup.py (hunk 2 above)
Comment (adiaaida) [reply scripts/ci_setup.py]: Yup.

File: scripts/dotnet.py (hunk 3 above)
Comment (adiaaida) [reply scripts/dotnet.py]: I'll fix. Copy paste

File: scripts/benchmarks_ci.py (hunk 1 above)
Comment (adiaaida) [reply scripts/benchmarks_ci.py]: Oh, right, extra delete!

File: scripts/ci_setup.py [hunk 4]
```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser, ArgumentTypeError
```
Comment (jorive) [inline scripts/ci_setup.py]: `ArgumentTypeError`, `datetime.datetime`, `platform` imported but not used.

File: scripts/ci_setup.py [hunk 5]
```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
```
Comment (jorive) [inline scripts/ci_setup.py]: If this is a stand anole script, could you please check it in as executable?

**Pull request merged**